bin/benchmark-in-docker.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Synopsis:
+# Benchmark the test runner Docker image.
+# Each run will clean the test directory being used to prevent previous runs from affecting the results.
+
+# Environment:
+# SKIP_DOCKER_BUILD: if set, skip building the Docker image
+
+# Output:
+# Outputs performance metrics.
+
+# Example:
+# ./bin/benchmark-in-docker.sh
+
+set -eo pipefail
+
+die() { echo "$*" >&2; exit 1; }
+
+required_tool() {
+    command -v "${1}" >/dev/null 2>&1 ||
+        die "${1} is required but not installed. Please install it and make sure it's in your PATH."
+}
+
+required_tool docker
+required_tool hyperfine
+
+# Pre-build the Docker image
+if [ -z "${SKIP_DOCKER_BUILD}" ]; then
```

Comment:
```suggestion
if [[ -z "${SKIP_DOCKER_BUILD}" ]]; then
```

Reply:
I mostly just copied these from the generic test runner so it might make sense to pr these changes into their as well: https://github.com/exercism/generic-test-runner/blob/main/bin/benchmark-in-docker.sh